lib/angular-barchart.js

```diff
@@ -3,9 +3,20 @@ angular.module('ftBarChart', [])
   return {
     restrict: 'A',
     scope: {
-      ngModel: '='
+      ngModel: '=',
+      objectKey: '@?barchartValueKey'
     },
     link: function(scope, elem, attrs) {
+      var dictMode = false;
+
+      if (angular.isObject(scope.ngModel[0])) {
```

Comment:
Not sure what's the benefit of that if you're suggesting something like `if (scope.ngModel.length && angular.isObject(scope.ngModel[0])) {` because an empty array can be checked for the first element (or anyone for this matter) and if it doesn't exist it will just be undefined, hence, not passing the test. 

If `ngModel` is not an array then the whole thing would blow up now and before. 

Thoughts?